deployments/arbitrum/usdc.n/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC(native)",
```

Comment:
Not sure if we should put `native` in the name.

Reply:
I have removed it :) 
(Thinking in follow-up PR will add (bridged) to the original market)